tests-e2e/fixture/k8s/fixture.go

```diff
@@ -0,0 +1,84 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/redhat-appstudio/managed-gitops/tests-e2e/fixture"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	. "github.com/onsi/gomega"
+
+	matcher "github.com/onsi/gomega/types"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+)
+
+// Create creates the given K8s resource, returning an error on failure, or nil otherwise.
+func Create(obj client.Object) error {
+
+	client, err := fixture.GetKubeClient()
```

Comment:
the variable `client` collides with imported package name `client "sigs.k8s.io/controller-runtime/pkg/client"`

```suggestion
	kubeClient, err := fixture.GetKubeClient()
```